doc/nrf/protocols/dect/index.rst

```diff
@@ -0,0 +1,144 @@
+
+Building upon the improvements over DECT, NR+ offers the following key features:
+
+License-Exempt Global Band
+   DECT NR+ utilizes the globally available 1.9 GHz band, which is license-free in most regions.
+   This reduces setup costs and complexity compared to LTE, which often requires licenses.
+
+Supported Channels
+   DECT NR+ operates within the 1.9 GHz band, using a specific channel bandwidth of 125 kHz.
+   This allows for denser network deployments compared to wider-bandwidth technologies.
+
+Modulation Schemes
+   DECT NR+ supports various modulation schemes like gaussian frequency-shift keying (GFSK) and differential quadrature phase shift keying (DQPSK), which allow selection based on the desired balance between data rate, range, and robustness in different operating conditions.
+
+Built-in Coexistence
+   DECT NR+ features the built-in coexistence of multiple networks in the same location, meaning that multiple DECT NR+ networks can operate in the same area without interference, enabling flexible deployment in dense environments.
+
+Flexible and Low-Latency network topologies
+   DECT NR+ supports various network configurations, like star networks or mesh networks, adapting to specific applications.
+
+High Reliability
+   DECT NR+ employs Automatic Repeat Request (ARQ) to correct data transmission errors, ensuring reliable communication for mMTC applications.
+
+Security
+   Possibility of hiding the network using AES-128 encryption and integrity protection.
+   DECT NR+ communication can be encrypted and the network itself hidden for enhanced security.
+
+High-speed
+   DECT NR+ reaches data rates up to 3.4 Mbps, depending on modulation.
+   DECT NR+ allows selecting the modulation scheme for high-speed communication suitable for various applications.
+
+Comparison of DECT NR+ and LTE
+******************************
+
+While both DECT NR+ and LTE address different needs, DECT NR+ offers distinct advantages in specific areas, as summarized by the following table:
+
+.. table:: Feature comparison of DECT NR+ and LTE
+
+   +---------------------------------------+-----------------------------------+----------------------------+
+   | Feature                               | DECT NR+                          | LTE                        |
+   +=======================================+===================================+============================+
+   | License                               | License-exempt 1.9 GHz            | Requires licenses (varies) |
+   +---------------------------------------+-----------------------------------+----------------------------+
+   | IP based                              | No                                | Yes                        |
+   +---------------------------------------+-----------------------------------+----------------------------+
+   | Coverage                              | Shorter, up to 3 km               | Wider, greater than 10 km  |
```

Comment:
I think it might be better to state that LTE (e.g. NB-IoT) coverage can be tens of kilometers (100 km has been demonstrated by Ericsson)